### PR TITLE
Add mysql support in fixtures

### PR DESCRIPTION
**Pull request by JustSkiv**
List of changes:

- postgres loader extracted to own package
- added common interface instead of specific loader
- added param db-type to specify the database type
    (only postgres is supported now)
- test-cases definition extracted to separate files from code
    (for more clarity and for shared access between tests for other DBs)
- added param DbType to RunWithTestingParams
- **added mysql support**

### CODE REVIEW

**Review by luza — APPROVED** [review]

File: fixtures/mysql/mysql_test.go [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+	t.Helper()
+
+	idCounter++
+
+	mock.ExpectExec(
+		fmt.Sprintf("^INSERT INTO `%s` %s VALUES %s$",
+			table,
+			fieldsToDbStr(fields),
+			valuesToInsert,
+			//valuesToDbStr(values),
```
Comment (What-If-I) [inline fixtures/mysql/mysql_test.go:207]: Should this really be commented?

**Review by What-If-I — CHANGES_REQUESTED** [review]
Everything looks fine except one small comment.

Thank you very much for your contribution and effort that you put into it!

**Comment by What-If-I** [comment]
@keyclaim, I believe you also wanted to have support for MySQL in Gonkey. Here it is 🙂

File: fixtures/mysql/mysql_test.go (hunk 1 above)
Comment (JustSkiv) [reply fixtures/mysql/mysql_test.go]: It's just a mess. Fixed, thank you.

**Comment by JustSkiv** [comment]
Thank you for your concern and review :slightly_smiling_face:

**Comment by keyclaim** [comment]
Great job!

**Pull request merged**